testrunner-running/pom.xml

```diff
@@ -41,6 +41,10 @@
             <groupId>xmlunit</groupId>
             <artifactId>xmlunit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-launcher</artifactId>
```

Comment:
Do you need this dependency declaration if you already have it in the parent pom?

Reply:
Yes, we need it. Otherwise, it fails to compile.

Reply:
Do you need it on the parent pom as well then?

Reply:
Since the parent does not have any code, it doesn't actually need any real dependency, but having them declared under dependencyManagement there is fine as to keep consistent versions throughout.